config/application.rb

```diff
@@ -28,5 +28,6 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+    config.autoload_paths << Rails.root.join("lib")
```

Comment:
Does this mean we can get rid of all the `require` statements elsewhere for stuff in here?

Reply:
Seemingly not...I've been struggling to understand why this became an issue in the first place, and the fact that despite this change we can't get rid of the require statements is even more baffling 🤔

Reply:
Same here. I don't see why tests like `spec/lib/google_analytics/client_spec.rb` ever passed. But I do know why the require statements are still necessary. It's to do with the way [autoloading works](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html): we're not autoloading anything under `lib/local-links-manager` because the directory name should actually be `lib/local_links_manager` to match Rails conventions for autoloading. If you change 'load_defaults` to `6.0`, it actually has this error specifically.

But I don't think any of that is in the scope of this PR.